app/controllers/concerns/account_brexit_checker_concern.rb

```diff
@@ -42,11 +40,11 @@ def pre_update_results
   end
 
   def logged_out_pre_saved_results_path
-    transition_checker_new_session_path(redirect_path: transition_checker_saved_results_path, _ga: params[:_ga])
+    transition_checker_new_session_url(redirect_path: transition_checker_saved_results_path, _ga: params[:_ga])
   end
 
   def logged_out_pre_update_results_path
-    transition_checker_new_session_path(redirect_path: transition_checker_save_results_confirm_path(c: criteria_keys), _ga: params[:_ga])
+    transition_checker_new_session_url(redirect_path: transition_checker_save_results_confirm_path(c: criteria_keys), _ga: params[:_ga])
```

Comment:
These methods only work if `SessionPathConcern` is also included in the controller.  This isn't great, because it means we have two separate modules which don't really work independently.  So I still think the `SessionPathConcern` should be merged into the existing concern(s).

If nothing else, having three separate concerns, all implementing different pieces of the same thing, and which are only used in a single controller, is a bit of a code smell.

Reply:
You make a compelling case! I have merged them.